### ISSUE TITLE
Does this work with ADFS servers?

### ISSUE THREAD

**Issue by MrJeric0** [zibasec/django-saml2-pro-auth#7]
Does this work with ADFS servers?

**Comment by defionscode** [comment]
Not yet. I have not added that functionality as I dont have an ADFS IdP to test with. However, if you look here https://github.com/MindPointGroup/django-saml2-pro-auth/blob/c06a316d18a08549e0f5af30e8456295efb08bd8/src/django_saml2_pro_auth/utils.py#L60-L63

Hypothetically, the only thing we would need is for that to be supported in settings.py and then we _should_ be good to go. It's really just an encoding thing. Is that something you'd feel comfortable adding and contributing back? Otherwise I can make a branch with support that you could then test out for me, however you wouldnt be able to install the branch via pip.

**Comment by jrial** [comment]
@defionscode actually, it's perfectly possible to install new branches through pip. E.g. if I wanted to install your `ampersand-esc` branch, I would type:
```
pip install -e git+git@github.com:MindPointGroup/django-saml2-pro-auth.git@ampersand-esc#egg=django-saml2-pro-auth
```

or alternatively, if I'd prefer to check out over HTTPS instead:
```
pip install -e git+https://github.com/MindPointGroup/django-saml2-pro-auth.git@ampersand-esc#egg=django-saml2-pro-auth
```

So yeah, it's feasible to publish testing branches for folks to try out. You just have to provide the instructions as most Python devs are not familiar with this feature.

ps: this notation also works in requirements.txt files. Just drop the `pip install ` from the start of the line, and paste the remainder into your requirements file. Handy for evaluating stuff that's not available through pip.

**Comment by shepdelacreme** [comment]
Fixed in #38
